packages/resolve-scripts/src/scripts/init.js

```diff
@@ -105,15 +106,52 @@ export default (appPath, appName, originalDirectory) => {
     const scriptsPath = path.join(appPath, 'node_modules', scriptsPackageName);
     const useYarn = fs.existsSync(path.join(appPath, 'yarn.lock'));
 
+    const appPackage = require(path.join(appPath, 'package.json'));
+    appPackage.scripts = {
+        build: 'resolve-scripts build',
+        dev: 'resolve-scripts dev',
+        start: 'resolve-scripts start',
+        test: 'jest',
+        'test:e2e': 'cross-env NODE_ENV=tests babel-node ./tests/testcafe_runner.js'
+    };
+    fs.writeFileSync(
+        path.join(appPath, 'package.json'),
+        JSON.stringify(appPackage, null, 2)
+    );
+
     const readmeIsExist = tryRenameReadme(appPath);
 
     const templatePath = path.join(scriptsPath, 'dist', 'template');
+
     if (!tryCopyTemplate(templatePath, appPath)) {
         error(`Could not locate supplied template: ${chalk.green(templatePath)}`);
         return;
     }
 
-    installDependencies(useYarn);
+    log('Installing app dependencies...');
+    log();
+
+    const dependencies = [
+        'axios',
```

Comment:
Maybe extract packages that need only for todolist in a separate array... Do something like that:
```
const appDependencies = ['axious', 'uuid'];
...
installDependencies(useYarn, appDependencies.concat(dependencies), false);
```